libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/Customization.kt

```diff
@@ -0,0 +1,59 @@
+package com.mapbox.navigation.dropin
+
+import com.mapbox.navigation.dropin.binder.UIBinder
+import com.mapbox.navigation.ui.maps.route.arrow.model.RouteArrowOptions
+import com.mapbox.navigation.ui.maps.route.line.model.MapboxRouteLineOptions
+
+/**
+ * DropInNavigationView customizations.
+ */
+class Customization {
+
+    /**
+     * Customize the speed limit view by providing your own [UIBinder].
+     * Use [UIBinder.USE_DEFAULT] to reset to default.
+     */
+    var speedLimit: UIBinder? = null
+
+    /**
+     * Customize the maneuvers view by providing your own [UIBinder].
+     * Use [UIBinder.USE_DEFAULT] to reset to default.
```

Comment:
Interesting `USE_DEFAULT`. Is there a way to determine the state of `DropInNavigationView`?

For example, what if we want to know all the current customizations?

Reply:
Just look at each StateFlow value in NavigationUIBinder and compare it with defaults.

Reply:
Yeah, my comment here is looking into a crystal ball. +1 to add `USE_DEFAULT`